inference-engine/src/vpu/graph_transformer/src/stages/gather_elements.cpp

```diff
@@ -127,4 +163,33 @@ void FrontEnd::parseGatherElements(const Model &model, const ie::CNNLayerPtr &la
                                           inputs[1], outputs[0], axis);
 }
 
+void FrontEnd::parseExpGatherElements(const Model &model, const ie::CNNLayerPtr &layer,
```

Comment:
I don't think you need `parseExpGatherElements`, just check the number of inputs in `parseGatherElements` and handle the difference.

Reply:
Fixed